modules/realvuBidAdapter.js

```diff
@@ -0,0 +1,55 @@
+import adaptermanager from 'src/adaptermanager';
+
+var utils = require('src/utils.js');
+var adloader = require('src/adloader.js');
+var bidmanager = require('src/bidmanager.js');
+var bidfactory = require('src/bidfactory.js');
+var Adapter = require('src/adapter.js').default;
+var AppnexusAdapter = require('./appnexusBidAdapter.js');
+
+var RealVuAdapter = function RealVuAdapter() {
+  var baseAdapter = new Adapter('realvu');
+  baseAdapter.callBids = function (params) {
+    // utils.logMessage('realvuBidAdapter params: ' + JSON.stringify(params));
+    var pbids = params.bids;
+    var boost_back = function() {
+      for (var i = 0; i < pbids.length; i++) {
+        var bid_rq = pbids[i];
+        var sizes = utils.parseSizesInput(bid_rq.sizes);
+        top.realvu_boost.addUnitById({
```

Comment:
References to `top` should be placed in a try/catch block to prevent possible uncaught `TypeError`s